api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -66,9 +62,9 @@ message PluginConfig {
   string root_id = 2;
 
   // Configuration for finding or starting VM.
-  oneof vm_config {
-    VmConfig inline_vm_config = 3;
-    // In the future add referential VM configurations.
+  oneof vm {
+    VmConfig vm_config = 3;
+    // TODO: add referential VM configurations.
```

Comment:
Why this changed from last API upstram?

Reply:
This change was never pulled into envoy-wasm and will require a change in istio.  Given that it isn't implemented in Envoy and is not going to be implemented before v4 I didn't think there was any value in causing code churn at this point for a feature which doesn't exist.  We can have a breaking change in v4 when we undoubtedly will have other breaking changes rather than introducing another one unnecessarily here.

Reply:
SGTM.